diff --git a/ModuleRadio/Firmware/src/ZiGate/Build/ZigbeeNodeControlBridge/Makefile b/ModuleRadio/Firmware/src/ZiGate/Build/ZigbeeNodeControlBridge/Makefile
--- a/ModuleRadio/Firmware/src/ZiGate/Build/ZigbeeNodeControlBridge/Makefile
+++ b/ModuleRadio/Firmware/src/ZiGate/Build/ZigbeeNodeControlBridge/Makefile
@@ -375,8 +375,6 @@
 	$(CC) -Wl,--gc-sections -Wl,-u_AppColdStart -Wl,-u_AppWarmStart $(LDFLAGS) -T$(ZNCLKCMD) -o $@ -Wl,--start-group $(APPOBJS) $(addprefix -l,$(LDLIBS))  -lm -Wl,--end-group -Wl,-Map,$(GENERATED_FILE_NAME).map
 
 	$(SIZE) $@
-	date +%c >> $(APP_BASE)/Doc/size.txt
-	$(SIZE) $@ >> $(APP_BASE)/Doc/size.txt
 
 # Do a second linking with LTO disabled
 	$(CC) -Wl,--cref -Wl,--gc-sections -Wl,-u_AppColdStart -Wl,-u_AppWarmStart $(LDFLAGS) -T$(ZNCLKCMD) -L $(SDK_BASE_DIR)/Stack/ZCL/Build/  -o $@ -Wl,--start-group $(APPOBJS) $(addprefix -l,$(LDLIBS)) -lm -Wl,--end-group -fno-lto -Wl,-Map,$(GENERATED_FILE_NAME)_NOLTO.map
@@ -405,4 +403,3 @@
 	rm -f $(APP_SRC_DIR)/pdum_gen.* $(APP_SRC_DIR)/zps_gen*.* $(APP_SRC_DIR)/pdum_apdu.*
 
 ###############################################################################
-
